packages/vscode-extension/src/debug/prerequisitesHandler.ts

```diff
@@ -149,6 +155,18 @@ async function checkPort(
 }
 
 export async function checkAndInstall(): Promise<Result<any, FxError>> {
+  // skip debugging if there is already a debug session
+  if (allRunningDebugSessions.size > 0) {
+    VsCodeLogInstance.warning("SKip debugging because there is already a debug session.");
+    return err(
+      returnUserError(
+        new Error("SKip debugging because there is already a debug session."),
```

Comment:
Will this error displayed in extension notification? If yes, maybe move it to strings.json for later localization.

Reply:
No, it will only shows in the output channel.